packages/web-app-files/src/components/FilesList/ContextActions.vue

```diff
@@ -36,23 +24,11 @@ export default {
   components: { ContextActionMenu },
   mixins: [
     FileActions,
-    AcceptShare,
-    Copy,
     CreateQuicklink,
-    DeclineShare,
-    Delete,
-    DownloadArchive,
-    DownloadFile,
     EmptyTrashBin,
-    Favorite,
-    Move,
     Paste,
-    Navigate,
-    Rename,
-    Restore,
     ShowActions,
     ShowDetails,
-    ShowEditTags,
```

Comment:
FYI: There were registered twice because they are already included in the generic `FileActions` mixin.